src/aotcompile.cpp

```diff
@@ -239,6 +331,113 @@ static void makeSafeName(GlobalObject &G)
         G.setName(StringRef(SafeName.data(), SafeName.size()));
 }
 
+bool isinlibjulia(std::string name) {
```

Comment:
I'm not quite sure what's going on here, but this and `jl_name_from_type` look fishy :-)

Reply:
By which I mean - having these lists written out makes me think "there must be a better way" ;-)